.yamllint.yml

```diff
@@ -26,5 +26,7 @@ rules:
   indentation:
     level: warning
     indent-sequences: consistent
+    ignore: |
+      .github/workflows/codeql-analysis.yml
```

Comment:
Let's just fix this? What's wrong with it?

Reply:
spacing. i dont think it is relevant